webpack.config.node.js

```diff
@@ -55,7 +55,7 @@ function getExternals() {
 
 const webpackConfig = {
 	devtool: 'source-map',
-	entry: 'index.js',
+	entry: path.join( __dirname, 'index.js' ),
```

Comment:
Agreed - honestly I assumed that was going to merge on friday and then i'd rebase this and remove it.